mxcube3/routes/SampleCentring.py

```diff
@@ -601,8 +601,29 @@ def accept_centring():
 
 @mxcube.route("/mxcube/api/v0.1/sampleview/centring/reject", methods=['PUT'])
 def reject_centring():
-    """
-    Reject the centring position.
-    """
+    """Reject the centring position."""
     mxcube.diffractometer.rejectCentring()
     return Response(status=200)
+
+
+@mxcube.route("/mxcube/api/v0.1/sampleview/moveto", methods=['PUT'])
+def pos_from_x_y():
```

Comment:
In order to see and visually analyse the sample, easier to find crystals where you do the centring, a quick way of moving a potential good position to an easier to manage image area.

On the other hand, implementing the _moveToBeam_ is different in:
* MiniDiff: basically what we have done in the PR but with beam-pos, the calc is the same, only with two motors.
```
            beam_xc = self.getBeamPosX()
            beam_yc = self.getBeamPosY()
            self.centringPhiz.moveRelative((y-beam_yc)/float(self.pixelsPerMmZ))
            self.centringPhiy.moveRelative((x-beam_xc)/float(self.pixelsPerMmY))
```
* GenericDiff: it calls ```get_centred_point_from_coord```, which gives you a centringPoint (?) and takes into account more motors (and raises _NotImplementedError_). But, Ivars already implemented it in EMBL diff so we could _steal_ the code which does all the calcs, but this will not be compatible with 2.1 I guess.

If you do not see the usefulness of a _move_to_center_ we can change to a _move_to_beam_ (it would have come sooner or later), and call to the code in the Minidiff (we would then implement those lines in our MD2/MD3 for now). Switching to the CentringMath hwobj approach would require a bit more of work. Would that be ok?